Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/PersonDetailsFragmentTest.kt

```diff
@@ -408,7 +408,7 @@ class PersonDetailsFragmentTest : BaseUITest() {
             every { fullName } returns "Andrea Schneider"
             every { uniqueCertificateIdentifier } returns "RN:UVCI:01:AT:858CC18CFCF5965EF82F60E493349AA5#K"
             every { dateOfBirthFormatted } returns "1981-03-20"
-            every { validUntil } returns Instant.parse("2021-03-31T11:35:00.000Z").toLocalDateUserTz()
+            every { testedPositiveOn } returns Instant.parse("2021-05-23T11:35:00.000Z").toLocalDateUserTz()
```

Comment:
`23.05.` is my birth day 😎

Reply:
@LukasLechnerDev  thats why it put it in ;-)